docs/docs/ports.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Ports
+---
+
+# Ports
+
+Port configuration for applications in Marathon can be confusing and there is [an outstanding issue](https://github.com/mesosphere/marathon/issues/670) to redesign the ports API. This page attempts to explain more clearly how they work.
+
+## Definitions
+
+*containerPort*: A _container port_ specifies a port within a container. This is only necessary as part of a _port mapping_ when using `BRIDGE` mode networking with a Docker container.
+
+*hostPort*: A _host port_ specifies a port on the host to bind to. This is only necessary as part of a _port mapping_ when using `BRIDGE` mode networking with a Docker container.
```

Comment:
"normal" non-docker ports are also really host ports. One of the important things is that this is how you other tasks can reach this task.

The host port is also then made available in the environment variables (and not the container port).